docs/examples/pytorch/pytorch-basic_example.ipynb

```diff
@@ -0,0 +1,151 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using DALI in PyTorch\n",
+    "\n",
+    "### Overview\n",
+    "\n",
+    "This example shows how to use DALI in pytorch.\n",
+    "\n",
+    "The CaffeReader reader is used in this example.\n",
+    "For details on how to use other data type please see other [examples](..)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us start from defining some global constants"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Caffe LMDB\n",
+    "lmdb_folder = \"/data/imagenet/train-lmdb-256x256\"\n",
+    "\n",
+    "N = 8             # number of GPUs\n",
+    "BATCH_SIZE = 128  # batch size per GPU\n",
+    "ITERATIONS = 32\n",
+    "IMAGE_SIZE = 3"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us define pipeline with the reader:"
```

Comment:
```suggestion
    "Let us define a pipeline with a reader:"
```
Additionally, the comment about using CaffeReader and refering to other examples could be moved here.

Reply:
Done